frontend/src/components/common/form/SearchToggle.tsx

```diff
@@ -0,0 +1,79 @@
+import { ReactComponent as MapSvg } from 'assets/images/icon-map.svg';
+import { ReactComponent as TableSvg } from 'assets/images/icon-table.svg';
+import React from 'react';
+import { useHistory } from 'react-router';
+import styled from 'styled-components';
+
+import TooltipWrapper from '../TooltipWrapper';
+import { ButtonProps } from '.';
+
+export enum SearchToggleOption {
+  /** The map is the active page */
+  Map = 'map',
+  /** Property List View is the active page */
+  List = 'list',
+}
+
+interface ISearchToggleProps extends ButtonProps {
+  /** set the id of the tool tip use for on hover of the plus buttons */
+  toolId: string;
+  /** Which toggle view is currently active */
+  toggle?: SearchToggleOption;
+}
+
+/**
+ * SearchToggle displaying two buttons which act like a toggle to navigate from the map to the property list view (and vise-versa).
+ * @param param0
+ */
+export const SearchToggle: React.FC<ISearchToggleProps> = ({
+  toggle = SearchToggleOption.Map,
+  toolId,
+  ...props
+}) => {
+  const history = useHistory();
+
+  return (
+    <StyledToggle>
+      <StyledNav>
+        <StyledLink style={{ cursor: toggle === SearchToggleOption.List ? 'pointer' : 'auto' }}>
```

Comment:
I'd recommend using a class to toggle these styles, That way you don't have to repeat the fill: toggle === SearchToggleOption.List conditional.
so: 
1) do the conditional **once** to set the class.
2) use the class (or absence of) to set the desired css attributes.